community/contributing/updating_the_class_reference.rst

```diff
@@ -160,6 +160,8 @@ Edit the file for your chosen class in ``doc/classes/`` to update the class refe
 
 Edit it using your favourite text editor. If you use a code editor, make sure that it doesn't change the indent style: tabs for the XML, and 4 spaces inside BBcode-style blocks. More on that below.
 
+If you need to check that the modifications you've made are correct in the generated documentation, build Godot as described :ref:`here <compiling>`, run the editor and open the help for the page you modified.
```

Comment:
The link to "compiling" doesn't seem to work in the generated HTML docs. How do I link to [this](https://docs.godotengine.org/en/3.1/development/compiling/) page?